### PR TITLE
Choice

### PR DESCRIPTION
**Pull request by Gicarr1998**
[Commands]
1. rails db:migrate:reset db:seed

[Pre-conditions]
1. Choices has no right answer yet

[Expected Output]
User can login as seeded user

[Notes]

### CODE REVIEW

File: choice/_form.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+Mauris in neque nisi. Aliquam id arcu ac arcu dictum rhoncus et sed erat.
+Quisque id pulvinar nibh. Proin volutpat turpis fermentum quam ornare, a venenatis nunc condimentum.
+Nam leo ligula, luctus at nisi vel, semper accumsan metus. Suspendisse potenti.
+Mauris purus erat, viverra tempor ante sit amet, porta iaculis odio.
```
Comment (Gicarr1998) [inline choice/_form.html:5]: Wrong indentions

File: word/new.html [hunk 2]
```diff
@@ -1 +1 @@
-new
\ No newline at end of file
+render 'form'
```
Comment (Gicarr1998) [inline word/new.html]: wrong folder please add the real folder.

**Review by Gicarr1998 — COMMENTED** [review]
Please fix the ff.

File: choice/_form.html (hunk 1 above)
Comment (aplazand20) [reply choice/_form.html:5]: Very good implementation

**Review by aplazand20 — APPROVED** [review]
I approve this PR

**Pull request merged**